lib/aggregate/aggregate_store.rb

```diff
@@ -114,6 +115,15 @@ def reload
       result
     end
 
+    def aggregate_attribute(name)
+      aggregate_attributes[name]
+    end
```

Comment:
As we just discussed, this should be symmetrical with the method below and only use the handler that it needs. You might want to put it back to start with `get_` therefore. (Sorry about that thrash!)